Twitch EventSub library/Twitch.EventSub_Websocket.csproj

```diff
@@ -15,12 +15,13 @@
         <License>https://github.com/GimliCZ/TwitchEventSub_Websocket/blob/master/LICENSE</License>
         <RepositoryUrl>https://github.com/GimliCZ/TwitchEventSub_Websocket</RepositoryUrl>
         <PackageLicenseExpression>MIT</PackageLicenseExpression>
-        <Version>1.1.5</Version>
+        <Version>1.1.6</Version>
```

Comment:
This update contains breaking changes due to interface changes and more significant changes to the underlying client. I would increase minor version instead of patch version to `1.2.0`.

Reply:
Done